website/src/components/external-link.js

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import { css } from '@emotion/core';
+import * as colors from '../colors';
+
+const linkStyles = css`
+  color: ${colors.light.primary};
+  text-decoration: none;
+
+  :hover {
+    text-decoration: underline;
+  }
+  :hover,
+  :active {
+    outline-width: 0;
+  }
+  :visited {
+    color: ${colors.light.primarySoft};
```

Comment:
is this necessary? AFAIK I asked design if Link needed the `:visited` prop and they said no 🤷‍♂

Reply:
I don't know, I haven't done the design review of the website yet. Thanks for the reminder!